### PR TITLE
fix jitter on name validation

### LINKED ISSUE #367: Jitter on deploy page when name validation happens
@digitalfishpond Can you fix this? You already had a solution for this, I think.

### PR DESCRIPTION
**Pull request by digitalfishpond**
fixes #367 

- container class created to increase bottom margin to allow for pattern validation (2 lines, gets squashed to following input field if not, and pushes rest of form down causing jitter)

- container class created for validation message and progress bar preventing jitter when progress bar appears

### CODE REVIEW

**Comment by digitalfishpond** [comment]
#367

**Comment by bryk** [comment]
@cheld @maciaszczykm Can you review?

**Comment by maciaszczykm** [comment]
I'll review.

**Comment by maciaszczykm** [comment]
LGTM, issue does no longer exist.

File: src/app/frontend/deploy/deployfromsettings.html [hunk 1]
```diff
@@ -17,20 +17,24 @@
...
-        already exists within namespace <i>{{ctrl.namespace}}</i>.</ng-message>
-      <ng-message when="pattern">Application name should contain only lowercase letters, numbers, and '-' between words
-      </ng-message>
-      <ng-message when="maxlength">Application name should have no more than 63 characters</ng-message>
-    </ng-messages>
+    <div class="kd-deploy-form-name-container">
+      <input ng-model="ctrl.name" name="name" namespace="ctrl.namespace" required ng-pattern="ctrl.namePattern"
+             ng-model-options="{ updateOn: 'default blur', debounce: { 'default': 500, 'blur': 0 } }" kd-unique-name
+             ng-maxlength="ctrl.nameMaxLength">
+      <div style="height: 2px">
```
Comment (bryk) [inline src/app/frontend/deploy/deployfromsettings.html]: Can you move this style to a CSS class? Basically, always have all your styles in CSS classes. You can reuse the 2px variable there.

**Comment by bryk** [comment]
Quick PTAL.

**Comment by digitalfishpond** [comment]
my bad. I had intended to, and the class was ready ;) PTAL

**Comment by bryk** [comment]
Awesome, thank you :) LGTM

**Pull request merged**